ci/dhis2-verify-app.yml

```diff
@@ -83,31 +83,20 @@ jobs:
     #          with:
     #              node-version: 12.x
     #
-    #        - uses: c-hive/gha-yarn-cache@v1
-    #        - run: yarn install --frozen-lockfile
-    #
-    #        - name: Install Cypress binary
-    #          run: yarn cypress install
-    #
```

Comment:
Not necessary. Cypress already does its own caching and installing. It's why I've removed the `install: false` below as well. Our own caching wasn't caching in a way that was compatible with what cypress expects so letting the action manage it itself seemed like the simplest solution.